source/_patterns/components/04-toast-notifications/00-toast-notification.hbs

```diff
@@ -0,0 +1,14 @@
+<div class="pf-toast alert {{ alertClass }} {{#if alertDismissible}}alert-dismissible{{/if}}" role="alert">
+{{#if alertDismissible}}
+  <button type="button" class="close" data-dismiss="alert" aria-label="Close">
```

Comment:
Close icon is a thing by itself on bootstrap:
http://v4-alpha.getbootstrap.com/utilities/close-icon/